system/secret-stores/templates/cluster-store.yaml

```diff
@@ -0,0 +1,139 @@
+{{- if eq .Values.secrets_provider "op" }}
+apiVersion: external-secrets.io/v1beta1
+kind: ClusterSecretStore
+metadata:
+  name: cluster-secret-store
+spec:
+  provider:
+    onepassword:
+      connectHost: http://onepassword-connect.1password.svc.cluster.local:8080
+      vaults:
+        k8s: 1
+      auth:
+        secretRef:
+          connectTokenSecretRef:
+            namespace: 1password
+            name: onepassword-token
+            key: token
+
+---
```

Comment:
Having to duplicate these is not ideal however 1Password does not support `/` in item names where as it is conventional to use them in other secret stores.